packages/feedback-service/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "feedback",
-  "description": "Feedback micro-service for One Platform",
+  "description": "Feedback Microservice for One Platform enables to manage feedback.",
   "opservice": {
     "className": "Feedback",
     "serviceName": "feedback"
   },
-  "version": "",
+  "version": "0.0.2",
   "keywords": [
     "web-components",
     "html"
```

Comment:
Add tags related to backend/graphql server. Web Components and html is not relevant to a microservice.